test/rules/typedef-whitespace/both/onespace/test.ts.lint

```diff
@@ -220,7 +220,7 @@ function withPreceedingSpaceNoSubsequentSpaceFn(a :number, b :number) :number {
     }
 }
 
-function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b : number) : number {
+function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b ?: number) : number {
```

Comment:
@adidahiya I don't mind if you want to do `b?: number`, but I should also be able to do `b ?: number`. There is truly a bug in that if I have it configured to require a space that it should allow the question mark next to the colon. You don't have to use this feature if you don't want. 

This has been a bug since November #811. This used to work back with tslint 3.3.0, and now I'm stuck on this version because no-one is fixing this. So I fixed it myself. Now it works like it used to. It isn't a new feature.